src/test/scala/com/microsoft/sqlserver/jdbc/spark/integration/SqlServerReadWriteTests.scala

```diff
@@ -0,0 +1,81 @@
+package com.microsoft.sqlserver.jdbc.spark.integration
```

Comment:
Tests are great additions. Would be good to have all regression test added as part of this framework. 
Should be a separate PR though, and this be kept at JDBC compact change.

Reply:
removed, will publish another PR later with the integration tests